src/store/error-store.js

```diff
@@ -0,0 +1,22 @@
+import { Store } from 'consus-core/flux';
+
+let error = null;
+
+class ErrorStore extends Store{
```

Comment:
Do we need to add the ErrorStore in this PR? I think this is being handled in #19